content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md

```diff
@@ -0,0 +1,101 @@
+---
+weight: 48
+title: 10.18.0.245
+layout: redirect
+---
+
+{{< c8y-admon-info >}}
+These release notes refer to build versions
+- Core: 10.18.0.245
+- UI: 10.18.0.164
+{{< /c8y-admon-info >}}
+
+### Fixes
+
+<table>
+<colgroup>
+<col style="width: 15%;">
+<col style="width:50%;">
+<col style="width: 10%;">
+<col style="width: 12%;">
+<col style="width: 13%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+
+<tr>
+<td>Cockpit</td>
+<td>If the Cockpit application was configured to remove all common tabs on group or device level, the button to add dashboards was also unintentionally removed. This has been addressed.</td>
+<td>MTM-56025</td>
+<td>10.18.0.163</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>The data export feature in the data explorer has been improved. Now only active data points are included. Moreover, the name of the exported file includes the name of the series. This file contains all measurements in which this series is found.</td>
+<td>MTM-50586</td>
+<td>10.18.0.162</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>An issue with module federation has been fixed. If the shell application was built on a windows machine the modules have not been shared properly between shell and plugin.</td>
+<td>MTM-56213</td>
+<td>10.18.0.161</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>Fixed an issue with SVG files in the SCADA widget, where <code>animate</code> or <code>animateTransform</code> tags worked fine in the configuration preview, but did not work in the dashboard view.</td>
+<td>MTM-55990</td>
+<td>10.18.0.160</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>In rare cases, when a real-time connection was interrupted, certain widgets, for example, did not update again once the connection was re-established. This behavior has been fixed.</td>

```

Comment:
i found real-time as well as real time (and realtime) in c8y-docs. which one is it?
also, lots of commas due to "for example" - could maybe made be fewer

Reply:
real-time is the adjective while realtime is the noun. I try to use it quite consistent, following this rule :-).